### PR TITLE
new tool to generate a custom-sized testnet (using ansible-playbook)

### PR DESCRIPTION
**Pull request by altergui**
> while trying to understand all the different components of the testsuite, i quickly wrote a small ansible playbook to flexibly recreate the files used by testsuite (docker-compose.yml, genesis.json and env files).

this ansible playbook turned out pretty useful to simulate different networks and helped me greatly into debugging `tendermint-0.35` branch.

definitely not the most traditional use of ansible, but i feel at home with jinja2 templating so for me it was the quickest way to solve this

### CODE REVIEW

**Comment by p4u** [comment]
Pretty cool! Can you review @vdo ?

File: cmd/dvotecli/commands/genesis.go [hunk 1]
```diff
@@ -125,12 +140,48 @@ func genesisGen(cmd *cobra.Command, args []string) error {
...
+	}
+
+	fmt.Println()
+
+	if file, _ := cmd.Flags().GetString("file"); file != "" {
+		if err := os.WriteFile(file, data.Bytes(), 0o600); err != nil {
+			return err
+		}
+	} else {
+		prettyHeader("Genesis JSON")
```
Comment (vdo) [inline cmd/dvotecli/commands/genesis.go]: Maybe we can avoid this header so the raw output JSON can be piped to tools like jq or ijq

**Comment by vdo** [comment]
Nice combination of ansible and docker compose :+1:

**Comment by p4u** [comment]
Can we automatize this testing somehow? Maybe a server running the test on each commit?

**Review by p4u — APPROVED** [review]

**Comment by p4u** [comment]
@altergui please address the comments and click "resolved" whenever is done :)

**Comment by p4u** [comment]
conflicts should be fixed :)

File: cmd/dvotecli/commands/genesis.go (hunk 1 above)
Comment (altergui) [reply cmd/dvotecli/commands/genesis.go]: nice one! done

**Comment by altergui** [comment]
> Can we automatize this testing somehow? Maybe a server running the test on each commit?

i thought about this but i want to avoid making the tests run even longer. I'll try a couple of alternatives and come up with a solution in another PR

**Pull request merged**